docs/en/sql-reference/aggregate-functions/parametric-functions.md

```diff
@@ -102,7 +104,7 @@ Events that occur at the same second may lay in the sequence in an undefined ord
 
 **Parameters**
 
-- `pattern` — Pattern string. See [Pattern syntax](#sequence-function-pattern-syntax).
```

Comment:
Quite a few of them from the list in the [issue](#https://github.com/ClickHouse/clickhouse-docs/issues/2389) were working, i changed them so we have some consistency in the docs. The custom tags are not needed because `#an example heading` can be found by `an-example-heading` without the need to add a custom tag like `anExampleHeading`